commanderConfig.js

```diff
@@ -19,10 +19,10 @@ module.exports = cmd.version('1.11.0','-v, --version')
     return s;
   },'1024x768')
   .option('-p, --provider [provider]', 'Set the image provider; '+Object.keys(img.list).join(', '),
-    function(p){
+    function setImageProvider(provider){
```

Comment:
```suggestion
    function(provider){
```
No need to name the function, we don't use it anywhere.